app/config/locale/terms.json

```diff
@@ -0,0 +1,1274 @@
+[
+    {
+        "term": "settings.inspire",
+        "context": "",
```

Comment:
The PR looks great, I would only think how can we make the metadata here much more useful for translator. Can we include a description? Maybe a highlighted screenshot with the relevant area in the UI once we translate the console?

Reply:
We can add information about a key in the "context" property like so 
 
![Screenshot from 2021-07-23 14-21-53](https://user-images.githubusercontent.com/20852629/126759026-6e2ec5cb-b3c3-41bc-bf15-bb09c09fa253.png)


We can attach images in the comments section like so 

![Screenshot from 2021-07-23 14-23-14](https://user-images.githubusercontent.com/20852629/126759151-fd8f2fa5-d3cf-4bef-952f-55c9f49278dc.png)

Reply:
Sounds good